pkg/acsengine/defaults_test.go

```diff
@@ -536,20 +578,34 @@ func getMockAddon(name string) api.KubernetesAddon {
 }
 
 func getMockBaseContainerService(orchestratorVersion string) api.ContainerService {
+	mockAPIProperties := getMockAPIProperties(orchestratorVersion)
```

Comment:
I don't believe any of the changes from here downwards are needed in this PR - is there any reason to keep them?

Reply:
I needed just `mockAPI` and we had just `MockBaseContainerService`, So to not to duplicate code I did this split. They are needed for other tests so this is for backwards compatibility for existing tests

Reply:
sorry, my bad - there's no reason to change the contents of the api.Properties at this stage, though?

Reply:
in general I don't think it's expected for more than one of orchestratorprofile.{kubernetes,dcos,openshift}config to be non-nil.  Also the masterprofile pieces may or may not be relevant to other users.  If it's not necessary for the test, please leave it out.  Please fix this and the comments and we'll finally merge.